cpu/numactl.py

```diff
@@ -70,7 +70,8 @@ def setUp(self):
 
     def test(self):
 
-        if build.make(self.sourcedir, extra_args='-k test', ignore_status=True):
+        if build.make(self.sourcedir, extra_args='-k -j 1'
```

Comment:
How about having the `extra_args` on the same line and `ignore_status` on the next line ?. Just a suggestion, if you do not want to change that's also fine.

Reply:
`pylint` and `pep8` is compliant with the existing way. I would like to keep it this way. Thanks

Reply:
It not about the compliance, I was mentioning about the readability. I am ok with it.